nautobot/core/templatetags/perms.py

```diff
@@ -28,3 +30,11 @@ def can_change(user, instance):
 @register.filter()
 def can_delete(user, instance):
     return _check_permission(user, instance, "delete")
+
+
+@register.filter()
+def can_cancel(user, instance):
+    if isinstance(instance, ApprovalWorkflow):
+        return instance.user == user and instance.is_active
+    else:
+        return _check_permission(user, instance, "change")
```

Comment:
Is this correct? What would be the case for `can_cancel` something that *isn't* an ApprovalWorkflow?